### DISCUSSION TITLE
Explosion to cause nearby people to fall on the ground?

### DISCUSSION THREAD

**Discussion by mefistotelis** [swfans/syndwarsfx#174] [General]
Now as I looked into the existing sprites and animation frames, I think there is material to prepare a falling down and standing up animation.

This could be used to make people fall if an explosion was nearby (or maybe only if a building was demolished?).

Also is there a properly implemented standing up animation? Like when knockout gas wears off?
If I remember right people are standing up instantly, without animation..

**Comment by Moburma** [comment]
> Also is there a properly implemented standing up animation? Like when knockout gas wears off?
> If I remember right people are standing up instantly, without animation..

They just instantly get up. I don't know about people falling over from explosions - they tend to just kill everyone. Getting up animations would be nice, although arguably annoying for your own characters. Would make sense for NPCs though, don't forget civilians often get run over by cars that they can recover from.
